pkg/kclient/operators.go

```diff
@@ -75,3 +76,20 @@ func (c *Client) SearchClusterServiceVersionList(name string) (*olm.ClusterServi
 		Items: result,
 	}, nil
 }
+
+// GetCSVWithCR returns the CSV (Operator) that contains the CR (service)
+func (c *Client) GetCSVWithCR(name string) (*olm.ClusterServiceVersion, error) {
```

Comment:
Unit test for this?

Reply:
I would really like to have them as well. I think I'll need to have a valid Operator definition for that. And I tried following [the documentation](https://docs.openshift.com/container-platform/4.5/operators/operator_sdk/osdk-getting-started.html#building-memcached-operator-using-osdk_osdk-getting-started) to create an Operator, but it's not trivial. If you have any ideas/suggestions to make it easy, let me know. Otherwise, I hate to admit this but, we will need to have to follow-up on this after v2. I can create an issue to cover that.

Reply:
Opened #3605